swarm/storage/localstore_test.go

```diff
@@ -0,0 +1,153 @@
+package storage
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/contracts/ens"
+)
+
+var (
+	hashfunc = MakeHashFunc("SHA3")
+)
+
+// convenience generator for unique chunks
+type randomChunkGenerator struct {
```

Comment:
We should refactor this soon - we have at least 3 different ways of instantiating unique random chunks, and we should unify them and not bother with this anymore.